src/browser/rvm/rvm_message_bus.ts

```diff
@@ -86,104 +215,127 @@ class RVMMessageBus extends EventEmitter  {
      *
      **/
     // TODO: the type of the data here is to be defined in RUN-2947
-    public send(topic: string, data: any, callback: Function, timeToLiveInSeconds: number) {
+    // public send(topic: string, data: any, callback: Function, timeToLiveInSeconds: number) {
```

Comment:
Its not yet 'defunct' per se, there are still parts of the code that use it I did not want to change over everything just yet. In a later PR this is removed